{{cookiecutter.python_name}}/.stylelintrc

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
@bollwyvl now that https://github.com/jupyterlab/jupyterlab/pull/11993 is in, should we simply copy-paste .stylelintrc.yaml from JupyterLab here to align configuration?